### DISCUSSION TITLE
5.0 stability and/or vague timeline?

### DISCUSSION THREAD

**Discussion by akanter** [ampache/ampache#2870] [General]
I've been very stoked about using ampache since the demise of Google Play Music but I have been patiently waiting for the 5.0 release because I didn't want to install 4.x and then have to reinstall for 5.0 shortly thereafter.

How stable would you rate 5.0 at least as far as the server installation goes? I'm OK if the client(s) are still buggy but as long as I don't have to mess with database migrations or mess around with installing different versions of PHP I'd love to get the ball rolling.

I also would love if there were some way for me to track how close we are to the 5.0 release - I'm not sure if https://github.com/ampache/ampache/projects/4?card_filter_query=milestone%3A5.0.0 is it or if there's somewhere else. How soon is it looking (vaguely)?

Thanks for all your hard work! I'm excited to start my new music life =)

**Comment by lachlan-00** [comment]
Most things aren't 100% for 5 but it's pretty much in prep mode now.

With all the work we did from December to march the develop branch is pretty stable though and I haven't had many issues that weren't also v4 bugs for a while.

If you don't want to do anything other than just roll with it develop is fine. When 5 actually releases you can reinstall from zip and use your config file and nothing will change

**Comment by lachlan-00** [comment]
@usox if I said release before the end of June do you think we're okay with that?

I know there's not a lot to get done but there's enough open features that it might take a bit.

**Reply by usox** [reply]
@lachlan-00 I'm fine with that. Maybe we should revise the 5.0 milestone feature list again to check, which features could be finished within april/may. Then at the end of may we should do a feature freeze for 5.0 to prepare for the release in june.
